mllib/src/main/scala/org/apache/spark/mllib/api/python/PythonMLLibAPI.scala

```diff
@@ -71,6 +71,14 @@ private[python] class PythonMLLibAPI extends Serializable {
       minPartitions: Int): JavaRDD[LabeledPoint] =
     MLUtils.loadLabeledPoints(jsc.sc, path, minPartitions)
 
+  def appendBias(data: org.apache.spark.mllib.linalg.Vector)
+  : org.apache.spark.mllib.linalg.Vector
```

Comment:
Scala style: Please follow [https://cwiki.apache.org/confluence/display/SPARK/Spark+Code+Style+Guide]

Also, is "Vector" not already imported?